README.md

```diff
@@ -1,145 +1,404 @@
-# Couchbase Connector for Streamlit
+# Couchbase-Streamlit Connector
```

Comment:
This should be something along the lines of Getting started/Building applications using Couchbase Streamlit Connector.